pypeit_test

```diff
@@ -108,9 +112,42 @@ def main():
         # Setups
         setups = next(os.walk(os.path.join(raw_data, instr)))[1]
 
+        # Cut me down
         if pargs.debug and ('blue' not in instr):
             continue
 
+        # Develop cut
+        if pargs.tests == 'develop':
+            if instr == 'Keck_DEIMOS':
+                req_setups = ['600ZD_M_600', '1200G_M_7750', '830G_LVM_8400', '830G_M_8500', '830G_L_8100']
+            elif instr == 'Gemini_GNIRS':
+                req_setups = ['GNIRS']
+            elif instr == 'Shane_Kast_blue':
+                req_setups = ['452_3306_d57', '600_4310_d55']
+            elif instr == 'Shane_Kast_red':
+                req_setups = ['600_7500_d55']
+            elif instr == 'Keck_NIRES':
+                req_setups = ['NIRES']
+            elif instr == 'Keck_LRIS_blue':
+                req_setups = ['long_400_3400_d560', 'long_600_4000_d560', 'multi_300_5000_d680', 'multi_600_4000_d560', 'multi_1200_340_d460']
```

Comment:
The LRIS multi 1200 setup does not run, because our LRIS reader cannot read the old blue side images before they output format changed (another incomplete port from LowRedux). There is thus no pypeit file. It would be nice to test our wavelengths for this setup, so we need someone to either fix the reader or find more recent test data for blue 1200.